feature-tests/tests/resources/page/context-page.ts

```diff
@@ -0,0 +1,20 @@
+import { WebDriver, By, until } from "selenium-webdriver";
+import { AbstractPageObject } from "./abstract-page-object";
+
+export class ContextPage extends AbstractPageObject {
+  private continueButton = By.xpath("//*[@id='start']");
+
+  constructor(driver: WebDriver) {
+    super(driver);
+  }
+
+  clickContinue = async () => {
+    await this.driver.wait(until.elementLocated(this.continueButton)), 2000;
```

Comment:
Should there be a bracket round the 2000 here?

Reply:
It should work either way